settings.py.example

```diff
@@ -1,15 +1,19 @@
 IRC = {
-    'SERVER': 'irc.twitch.tv',
-    'NICK': 'twitch_username',
-    'PASSWORD': 'your_oauth_password',  # tip: http://twitchapps.com/tmi/
+    'SERVER': 'irc.chat.twitch.tv',
+    'NICK': 'USERNAME',
+    'PASSWORD': 'PASSWORD',
```

Comment:
Is the format no longer oauth? Please add a comment saying which format is expected. Thanks!